cylc/flow/rundb.py

```diff
@@ -647,11 +647,12 @@ def select_jobs_for_restart(self, callback):
             SELECT
                 %(task_pool)s.cycle,
                 %(task_pool)s.name,
-                %(task_pool)s.status,
-                %(task_states)s.submit_num,
+                %(task_jobs)s.submit_num,
                 %(task_jobs)s.time_submit,
+                %(task_jobs)s.submit_status,
                 %(task_jobs)s.time_run,
                 %(task_jobs)s.time_run_exit,
+                %(task_jobs)s.run_status,
                 %(task_jobs)s.job_runner_name,
```

Comment:
The docstring is now out of date: https://github.com/cylc/cylc-flow/blob/6613349ed1069ef08b7512c81e29b23b4d4e9731/cylc/flow/rundb.py#L642-L645

Possibly just remove that list in the docstring as it's duplication of what can be read from the SQL query anyway